terraform/aws/modules/provisioner/providers.tf

```diff
@@ -0,0 +1,21 @@
+provider "kubernetes" {
+  host                   = data.aws_eks_cluster.cluster.endpoint
+  cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
+  token                  = data.aws_eks_cluster_auth.cluster_auth.token
+  load_config_file       = false
+  # config_path            = "${var.kubeconfig_dir}/kubeconfig"
```

Comment:
do we need the config path? as we load the config on the fly

Reply:
if we don't use the config path it means that when we run locally we always need to authenticate first right?